tests/crypto/test_keyring.py

```diff
@@ -465,15 +465,11 @@ async def get_json(destination: str, path: str, **kwargs: Any) -> JsonDict:
         res_keys = key_json[lookup_tuple]
         self.assertEqual(len(res_keys), 1)
         res = res_keys[0]
-        self.assertEqual(res["key_id"], testverifykey_id)
-        self.assertEqual(res["from_server"], SERVER_NAME)
```

Comment:
Why have these two gone, sorry? I think we removed `from_server` in the second commit, not sure about key_id though.

Reply:
Because `key_id` is not in the new struct, because it's the dictionary's key